### ISSUE TITLE
Auto-animate functionality problems

### ISSUE THREAD

**Issue by jcmoral90** [MSzturc/obsidian-advanced-slides#61]
I've had problems with the auto-animate functionality, with the new updates I no longer have the flashy effect. I remember @MSzturc posted an example [Example here](https://forum.obsidian.md/t/advanced-slides-create-markdown-based-reveal-js-presentations-in-obsidian/28243/39m). However, that example doesn't work for me either. Anyone else with this problem?

```
<!-- .slide: data-auto-animate -->
# Title 1 :  

---
<!-- .slide: data-auto-animate -->

# Title 1 :
##### **Subtitle 2**
###### *Author - 2022* 
---
```

**Comment by MSzturc** [comment]
For me it works as expected ( with 1.7.1 )

https://user-images.githubusercontent.com/4275753/162247482-a3b8f404-6a7c-4f2d-9f02-b1a19e57e437.mov

Can you describe the flashy effect you are speaking of?

**Comment by jcmoral90** [comment]
Hi @MSzturc , the effect now is different from the previous versions, can it be configured? For example on the revealjs page I would like it to look like "Implicit Animation". https://revealjs.com/auto-animate/

**Comment by MSzturc** [comment]
Ahhh you're right, the data-auto-animate property gets lost. Will be fixed in next release

**Comment by MSzturc** [comment]
fixed with 1.8.1
